src/pages/about.js

```diff
@@ -1,13 +1,289 @@
 import React from 'react'
-
+import styled from 'styled-components'
 import SEO from '../components/containers/seo'
 import Layout from '../components/layout/layout'
 import { localize, WithIntl } from '../components/localization'
+import Hero from '../components/elements/hero.js'
+import Container from '../components/containers/container.js'
+import Card from '../components/elements/card.js'
+
+import DubaiSVG from 'images/svg/dubai.svg'
+import KualaLumpurSVG from 'images/svg/kuala-lumpur.svg'
+import LabuanSVG from 'images/svg/labuan.svg'
+import MaltaSVG from 'images/svg/malta.svg'
+import ParaguaySVG from 'images/svg/paraguay.svg'
+import Button from '../components/form/button.js'
+import TradingAccountsOpenedSVG from 'images/svg/trading-accounts-opened.svg'
+import TradeLastMonthSVG from 'images/svg/trades-last-month.svg'
+import TotalTradeTurnoverSVG from 'images/svg/total-trade-turnover.svg'
+import VisionSVG from 'images/svg/vision.svg'
+import FuseeSVG from 'images/svg/fusee.svg'
+
+const OurLocationsWrapper = styled.div`
+    width: 100%;
+    background-color: --color-white;
+    text-align: center;
+    padding: 8rem 0;
+
+    h1 {
+        height: 60px;
+        font-size: 48px;
+        font-weight: bold;
+        line-height: 1.25;
+        color: #2a2a2a;
+    }
+    button {
+        margin-top: 4rem;
+        width: 30rem;
+        height: 4.8rem;
+        font-size: 16px;
+    }
+`
+const OurLocationsContainer = styled(Container)`
+    margin: 0 auto;
+    padding: 4rem 0 0 0;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: auto;
+    grid-column-gap: 20px;
+    grid-template-areas:
+        'kll kll kll kll dl dl dl dl pl pl pl pl'
+        '. . ml ml ml ml ll ll ll ll . .';
+`
+
+const Location = styled.div`
+    grid-area: ${props => props.location};
+
+    h3 {
+        margin-top: 0.5rem;
+        font-size: 20px;
+        line-height: 1.5;
+        color: #2a2a2a;
+    }
+`
+
+const OurLocations = () => {
+    return (
+        <OurLocationsWrapper>
+            <h1>Our Locations</h1>
+            <OurLocationsContainer>
+                <Location location="kll">
+                    <KualaLumpurSVG />
+                    <h3>
+                        Kuala Lumpur,
```

Comment:
import localize and localize all the text please

```suggestion
                        {localize('Kuala Lumpur')},
```